packages/validation/src/utils/joi-to-error-extensions.ts

```diff
@@ -94,6 +94,27 @@ export const joiValidationErrorItemToErrorExtensions = (
 		extensions.invalid = validationErrorItem.context?.value;
 	}
 
+	// TODO Find a better way of passing the expected value down to the client
+	if (joiType.endsWith('.pattern.name')) {
```

Comment:
```suggestion
	if (joiType.endsWith('.pattern.name') || joiType.endsWith('.pattern.invert.name') {
```

this is due to when the `invert` option is set to true:

https://github.com/directus/directus/blob/b211a3b68d4af52237decc68fc556941fb5935ec/packages/utils/shared/generate-joi.ts#L193-L196

the name ends up having `.invert` in between, as seen in https://joi.dev/api/?v=17.9.1#stringpatterninvertname.